source/contracts/Controller.sol

```diff
@@ -87,8 +88,8 @@ contract Controller is IController {
      * Contract Administration [dev mode can use it]
      */
 
-    function suicide(IControlled _target, address _destination) public devModeOwnerOnly returns(bool) {
-        _target.suicideFunds(_destination);
+    function suicide(IControlled _target, address _destination, IUniverse _universe) public devModeOwnerOnly returns(bool) {
```

Comment:
Needs rebase on/merge of develop, `suicide` was renamed to address Solidity compiler warning about function name conflicting with name of reserved word.

Reply:
What was it renamed to? I don't see it renamed anywhere

Reply:
@adrake33 You merged your change that renamed `suicide` right?  Or is that still pending merge today?